packages/components/src/color-palette/utils.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import type { RefObject } from 'react';
+import { colord, extend } from 'colord';
+import namesPlugin from 'colord/plugins/names';
+import a11yPlugin from 'colord/plugins/a11y';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import type { ColorObject, ColorPaletteProps, PaletteObject } from './types';
+
+extend( [ namesPlugin, a11yPlugin ] );
```

Comment:
I had to add `colord` extensions here as well, since without it the unit tests would fail. I don't think that this should be problematic, but better to double-check too